src/app/project/[id]/page.tsx

```
@@ -0,0 +1,11 @@
+'use client'
+
+import useProject from '@/hooks/useProject'
+import { useParams } from 'next/navigation'
+
+export default function Project() {
+  const params = useParams<{ id: string }>()
+  const project = useProject(Number(params.id))
```

ESLint

[error] 'project' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)